src/UserService.ts

```diff
@@ -132,6 +134,67 @@ class UserService extends Base {
       },
     );
   }
+
+  async activateSubscription({ username, license }: { username: string; license: License }): Promise<License> {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/users/${username}/licenses`;
+    const json = await throwExpected(
+      async () => fetch.post(url, license),
+      {
+        404: error => {
+          const message = error?.body.message;
+
+          if (typeof message === "string") {
+            if (message.includes("User")) {
+              return new UserNameNotFoundException(username);
+            }
+
+            return new NotFoundException(`Recurly subscription ${license.subscriptionId} not found`);
+          }
+
+          return new LensSDKException(
+            500,
```

Comment:
We should use the correct error code to make debugging easier.

Reply:
Added required statuses